src/odemis/gui/comp/legend.py

```diff
@@ -307,6 +385,20 @@ def on_size(self, _=None):
         if self._value_range:
             self.Refresh()
 
+        if self._orientation == wx.HORIZONTAL:
+            self.lockBtn.SetPosition((self.ClientSize.x - 24, 0))
+        else:
+            self.lockBtn.SetPosition((self.ClientSize.x - self.ClientSize.x / 2 - 12, 0))
+
+    def on_mouse_enter(self, _=None):
```

Comment:
Is it really needed to update the cursor every time the mouse enters/leaves? IIRC, with wxPython, SetCursor is local to the widget any way. So you could just change the cursor at init, and be done.